### ISSUE TITLE
Typing: ms with `StringValue` should not accept options

### ISSUE THREAD

**Issue by meshuamam** [vercel/ms#197]
Currently, the definition of `ms` in the typing is:
```
declare function ms(value: StringValue, options?: Options): number;
declare function ms(value: number, options?: Options): string;
```

the `StringValue` version should not accept `options`, since it doesn't use it

**Comment by mrmckeb** [comment]
Great catch @meshuamam. Would you be interested in raising a PR for that change?

**Comment by dimitropoulos** [comment]
this is true today, but it's an optional argument and we want to leave the door open to other formatting options (see currently open PRs, for example).  since the `options?` argument is already there, it's a forwards-compatible change to add an option that may affect parsing (i.e. instead of formatting, which currently is the only thing that uses the option).

we're working on finally releasing 3.0 - and looking at the in-progress PRs, I suspect we'll have more options soon.  but again, even if not, let's keep the door open.

that all said - really appreciate your attention to detail.  as we publish a release candidate for 3.0, would love to hear your feedback!  stay tuned!
